lib/services/github.ts

```diff
@@ -185,19 +185,16 @@ export class GithubService extends WorkerClient<string> implements ServiceListen
 				return false;
 			}
 
-			// Standard Express installation.
-			const app = express();
-			app.use(bodyParser.urlencoded({ extended: true }));
-			app.use(bodyParser.json());
+			const app = this.massageToExpress(listenerConstructor.server);
```

Comment:
Needs a comment, as was here previously, to signal intention. Name's not great, how about something like `getExpressInstance`?